packages/js/core/src/types/Client.ts

```diff
@@ -20,6 +21,7 @@ export interface ClientConfig<TUri extends Uri | string = string> {
   interfaces: InterfaceImplementations<TUri>[];
   envs: Env<TUri>[];
   uriResolvers: UriResolver[];
+  wrapperCache?: WrapperCache;
```

Comment:
Thoughts on naming this "cache"?

Reply:
Everything LGTM!

Reply:
I like the explicit nature of "wrapper cache", as there may be other things we want to cache in the future.